
class InternalError(Exception):
    """Internal error."""
    pass
